Changelog.md

```diff
@@ -20,6 +20,7 @@ Compiler Features:
 Bugfixes:
  * SMTChecker: Fix error that reports invalid number of verified checks for BMC and CHC engines.
  * SMTChecker: Fix formatting of unary minus expressions in invariants.
+ * SMTChecker: Fix internal compiler error when assigning function marked pure or view to a function type variable.
```

Comment:
```suggestion
 * SMTChecker: Fix SMT logic error when assigning function marked pure or view to a function type variable.
```

Shouldn't it be `SMT logic error`? I read https://github.com/ethereum/solidity/issues/14999 and from what I understood, the compiler considers both types the same, but the smt was seeing them as different, right?